spring-cloud-starter-task-composedtaskrunner/pom.xml

```diff
@@ -15,6 +15,7 @@
 
     <properties>
         <spring-cloud-common-security-config.version>1.1.3.RELEASE</spring-cloud-common-security-config.version>
+        <spring-security.version>5.2.0.RC1</spring-security.version>
```

Comment:
Well, above `spring-cloud-common-security-config.version` is wrong.

Reply:
No, I think that version is correct. The app starters are still on Boot 2.1 etc. Maybe @cppwfs can clarify.